src/SSVOpenHexagon/Core/HGUpdate.cpp

```diff
@@ -27,9 +27,18 @@ void HexagonGame::update(ssvu::FT mFT)
 
     // TODO: refactor to avoid repetition, and truncate floating point number
     // TODO: also show best record (here) and last run + best record (in menu)
-    steamManager.set_rich_presence_in_game(
-        levelData->name, status.getTimeSeconds());
-    steamManager.run_callbacks();
+
+    const int DELAY_TO_UPDATE = 5; // X seconds
+    static float timeTillUpdate = 0.0;
```

Comment:
This should be a data member of `HexagonGame` rather than a `static` variable. How about also calling it `timeUntilRichPresenceUpdate`?